charts/longhorn/104.3.1+up1.8.2/Chart.yaml

```diff
@@ -0,0 +1,39 @@
+annotations:
+  catalog.cattle.io/auto-install: longhorn-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Longhorn
+  catalog.cattle.io/kube-version: '>= 1.25.0-0'
+  catalog.cattle.io/namespace: longhorn-system
+  catalog.cattle.io/permits-os: linux,windows
+  catalog.cattle.io/provides-gvr: longhorn.io/v1beta1
+  catalog.cattle.io/rancher-version: '>= 2.9.0-0 < 2.10.0-0'
+  catalog.cattle.io/release-name: longhorn
+  catalog.cattle.io/type: cluster-tool
+  catalog.cattle.io/upstream-version: v1.8.2
```

Comment:
In the past this field usually does not have character `v` For example, [catalog.cattle.io/upstream-version: 1.8.1](https://github.com/rancher/charts/blob/24bb95c72075fdc04f7c493079ec66d71912abe9/charts/epinio/103.0.0%2Bup1.8.1/Chart.yaml#L13) Should we make it consistent?

Reply:
Updated. Thanks.